synapse/handlers/room_member.py

```diff
@@ -31,6 +28,10 @@
 from synapse.util.async import Linearizer
 from synapse.util.distributor import user_left_room, user_joined_room
 
+import abc
```

Comment:
https://github.com/matrix-org/synapse/blob/master/docs/code_style.rst:

> As per PEP-8, imports should be grouped in the following order, with a blank line between each group:
>
> standard library imports
> related third party imports
> local application/library specific imports

this is going in the opposite direction!

Reply:
Hmm. I thought we always put e.g. `import logging` last tbh. (Doesn't your IDE rearrange imports into that order?)

Reply:
it does not.